packages.md

```diff
@@ -4,7 +4,40 @@ This page contains packages recommended by HelloFresh Golang Clan. These are not
 
 If you need more packages or just for inspiration, visit [Awesome Go](https://awesome-go.com/).
 
+## Configuration
+
+- [github.com/kelseyhightower/envconfig](https://github.com/kelseyhightower/envconfig) - get configs from environment variables
+
 ## Database
 
 - [database/sql](https://golang.org/pkg/database/sql/) - standard SQL library that you should stick to or build upon
-- https://github.com/jmoiron/sqlx - extension to standard SQL library that makes it a bit more convenient
+- [github.com/jmoiron/sqlx](https://github.com/jmoiron/sqlx) - extension to standard SQL library that makes it a bit more convenient
+- [github.com/mattes/migrate](https://github.com/mattes/migrate) - database migrations
+
+## Forms
+
+- [github.com/gorilla/schema](https://github.com/gorilla/schema) - decode forms and urls into Go structs and vice versa
+
+## HTTP
+
+- [github.com/go-chi/chi](https://github.com/go-chi/chi) - router with decent performance and middleware support
+- [github.com/dgrijalva/jwt-go](https://github.com/dgrijalva/jwt-go) - JSON Web Token library
```

Comment:
Please use https://github.com/hellofresh/jwt-auth-go here - it wraps `github.com/dgrijalva/jwt-go` with HF-specific stuff